src/components/views/ProductList/ProductList.js

```diff
@@ -1,9 +1,58 @@
 import React from 'react';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import styles from './ProductList.module.scss';
+import ProductBox from '../../common/ProductBox/ProductBox';
+import PageNotFound from '../../views/PageNotFound/PageNotFound';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
 
-const ProductList = () => <div className={styles.root}>This is ProductList</div>;
+const ProductList = ({ id, name, products, error }) => {
+  if (error) return <PageNotFound />;
+  else
+    return (
+      <div>
+        <div className='navbar'>
+          <h2>{name}</h2>
+        </div>
+        <div>
+          <ul>
+            <li>
+              <a href='/'>Home</a>
+            </li>
+          </ul>
+        </div>
+        <div className='container'>
+          <div className='col-9'>
+            <div className='row'>
+              {products &&
+                products
```

Comment:
tutaj możesz dodać, że jeżeli id jest równe furniture to pokazujesz wszystkie meble, a jeżeli nie to filtrujesz po kategoriach.